### PR TITLE
Sorted bug issue template to improve link slack preview

### PR DESCRIPTION
**Pull request by evertonlperes**
### Changes
1. Sorted some fields in order to improve Slack link preview

### CODE REVIEW

**Comment by jandubois** [comment]
Why no changes to move the pre-flight checklist in the feature request template?

Also, is there a way to do the checkbox, so it doesn't count as a "task"? I dislike that all feature requests are listed with "1 task done".

**Comment by evertonlperes** [comment]
> Why no changes to move the pre-flight checklist in the feature request template?
> 
> Also, is there a way to do the checkbox, so it doesn't count as a "task"? I dislike that all feature requests are listed with "1 task done".

I don't like it either, but this is an embedded feature on github templates. I've seen people giving some feedback to Github (see [here](https://github.com/github/feedback/discussions/5197#discussioncomment-1232403)), but I didn't find any PR to fix that. 
In order to keep my mental sanity xD, I just ignore the `task` message on the feature request... I prefer to worry about it on Epics cards.

We can either remove the pre-flight checkbox (I think it's useful and it avoid people to open duplicated features) or leave it as is.. it's up to you.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -26,4 +19,11 @@ body:
     label: Additional Information
     description: Add any other context/information about the problem here.
   validations:
-    required: false
\ No newline at end of file
+    required: false
+- type: checkboxes
+  attributes:
+    label: Preflight Checklist
+    description: Please ensure you've completed the following.
```
Comment (ericpromislow) [inline .github/ISSUE_TEMPLATE/feature_request.yml:26]: "following." would be better as "following:"

The rest looks fine.

**Review by ericpromislow — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 1 above)
Comment (evertonlperes) [reply .github/ISSUE_TEMPLATE/feature_request.yml:26]: Thanks for catching!

**Review by gaktive — APPROVED** [review]
LGTM

**Pull request merged**